src/ops/fuelup_show.rs

```diff
@@ -17,33 +18,6 @@ use crate::{
     toolchain_override::ToolchainOverride,
 };
 
-fn exec_version(component_executable: &Path) -> Result<Version> {
-    match std::process::Command::new(component_executable)
-        .arg("--version")
-        .output()
-    {
-        Ok(o) => {
-            let output = String::from_utf8_lossy(&o.stdout).into_owned();
-            match output.split_whitespace().last() {
-                Some(v) => {
-                    let version = Version::parse(v)?;
-                    Ok(version)
-                }
-                None => {
-                    bail!("Error getting version string");
-                }
-            }
-        }
-        Err(e) => {
-            if component_executable.exists() {
-                bail!("execution error - {}", e);
```

Comment:
This is still a useful error. Could we incorporate this error handling into `get_bin_version` and have it return a Result instead of Option?